content/java/concepts/stringbuilder/terms/length/length.md

```diff
@@ -0,0 +1,52 @@
+---
+Title: '.length()'
+Description: 'Returns the number of characters currently in the StringBuilder.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Characters'
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.length()`** method returns the number of characters currently in the `StringBuilder`.
```

Comment:
I would include an additional sentence that explains how `.length()` (total characters _in use_) is different from `.capacity()` (total characters _available for use_).